src/main/java/tech/aroma/data/GuicePlus.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 RedRoma, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tech.aroma.data
+
+import com.google.inject.*
+import com.google.inject.binder.AnnotatedBindingBuilder
+import com.google.inject.binder.ScopedBindingBuilder
+
+/*
```

Comment:
At some point I should pull this out into an Alchemy-Kotlin library, because this is definitely not Kotlin-specific.